warehouse/ops-tools/index-validation/src/main/java/datawave/index/validation/AccumuloIndexAgeDisplay.java

```diff
@@ -11,11 +11,7 @@
 import org.apache.accumulo.core.data.Range;
 import org.apache.accumulo.core.data.Value;
 import org.apache.accumulo.core.security.Authorizations;
-import org.apache.commons.cli.BasicParser;
-import org.apache.commons.cli.CommandLine;
-import org.apache.commons.cli.CommandLineParser;
-import org.apache.commons.cli.Options;
-import org.apache.commons.cli.ParseException;
+import org.apache.commons.cli.*;
```

Comment:
A minor nit, but the reigning view on wildcard imports for DW is to avoid them.
Other than that, this PR is good to go